doc/whats_new/upcoming_changes/sklearn.ensemble/30833.feature.rst

```diff
@@ -0,0 +1,4 @@
+- :class:`ensemble.BaggingClassifier` and :class:`ensemble.BaggingRegressor` now support
+  metadata routing through their `predict`, `predict_proba`, `predict_log_proba` and
+  `decision_function` methods and pass `**predict_params` to the underlying estimators.
+  By :user:`Stefanie Senger <StefanieSenger>`.
```

Comment:
should this not go into the metadata routing section instead of `sklearn.ensemble`?